backend/db/refresh.py

```diff
@@ -97,18 +97,20 @@ def refresh_db(
     """
     local = use_local_db
     print('INFO: Starting database refresh.', flush=True)  # flush: for gh action
+
+    # Calculate: Start & end time
+    # end_time: Even though in reality the refresh will not end 1 microsecond after the start time, we're setting it
+    # this way because this is the easiest way to make sure that future refreshes will not miss any new data that was
+    # added between when the refresh started and ended.
     t0, start_time = datetime.now(), current_datetime()
+    end_time_reported: str = tz_datetime_str(dp.parse(start_time) + timedelta(microseconds=1))
```

Comment:
## Clarification: `end_time`
The `end_time` reported for the refresh is actually the same as the time when it started. This is to ensure that no data is missed the next time the refresh runs.

I updated the variable name, the position where it is declared, and the code comments in order to make this less confusing.